libs/sdk-ui-filters/src/AttributeFilter/hooks/useAttributeFilterController.ts

```diff
@@ -572,6 +574,9 @@ function updateNonResettingFilter(
                 handler.setDisplayForm(displayFormRef);
                 handler.setDisplayAsLabel(displayAsLabel);
             }
+            if (enableDashboardFiltersApplyModes) {
+                handler.setSearch("");
```

Comment:
This is potentially incompatible change. We clear filter search after filter from props has been changed. e.g. applied.